src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+body {
+  margin-top: 0;
+  box-sizing: border-box;
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  position: relative;
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search::placeholder {
+  background-position: center left 20px;
+}
```

Comment:
use input's paddings to position a placecholder and add correct font styles to the placeholder